akka-http-caching/src/main/scala/akka/http/routing/directives/CachingDirectives.scala

```diff
@@ -0,0 +1,92 @@
+package akka.http.routing.directives
```

Comment:
The package for the Scala directives should be `akka.http.scaladsl.server.directives`